server/resource_server/repository/mem_repository.go

```diff
@@ -408,3 +408,11 @@ func (r *MemoryRepository) SaveZkSnarksKeyPair(keyPair models.ZkSnarksKeyPair) (
 func (r *MemoryRepository) GetLatestZkSnarksKeys() (models.ZkSnarksKeyPair, error) {
 	return models.ZkSnarksKeyPair{}, nil
 }
+
+func (r *MemoryRepository) GetUserForUsername(username string) (models.User, error) {
+	return models.User{}, nil
```

Comment:
Nice work around. I was wondering how you'd get around this.

Reply:
Yeah, since this memory repository is legacy, we can just stub the new repository methods